src/main/java/io/mosip/digitalcard/util/TemplateGenerator.java

```diff
@@ -72,29 +75,31 @@ public class TemplateGenerator {
 	 *             Signals that an I/O exception has occurred.
 	 *             the apis resource access exception
 	 */
-	public InputStream getTemplate(String templateTypeCode, Map<String, Object> attributes, String langCode)
+	public InputStream getTemplate(String cardTemplate, Map<String, Object> attributes, String langCode)
 			throws Exception {
 
-		ResponseWrapper<?> responseWrapper;
-		TemplateResponseDto template;
+		//ResponseWrapper<?> responseWrapper;
+		//TemplateResponseDto template;
 		printLogger.debug("TemplateGenerator::getTemplate()::entry");
 
 		try {
-			List<String> pathSegments = new ArrayList<>();
+		/*	List<String> pathSegments = new ArrayList<>();
 			pathSegments.add(langCode);
 			pathSegments.add(templateTypeCode);
 
 			responseWrapper = (ResponseWrapper<?>) restClient.getApi(ApiName.TEMPLATES, pathSegments, "", "",
 					ResponseWrapper.class);
 			template = mapper.readValue(mapper.writeValueAsString(responseWrapper.getResponse()),
 					TemplateResponseDto.class);
-
 			InputStream fileTextStream = null;
 			if (template != null) {
 				InputStream stream = new ByteArrayInputStream(
 						template.getTemplates().iterator().next().getFileText().getBytes());
 				fileTextStream = getTemplateManager().merge(stream, attributes);

```

Comment:
Don't comment, remove all the unwanted code.

Reply:
taken care